src/main/java/org/mifos/connector/channel/gsmaapi/GSMADepositApiController.java

```diff
@@ -0,0 +1,30 @@
+package org.mifos.connector.channel.gsmaapi;
+
+import com.fasterxml.jackson.core.*;
+import com.fasterxml.jackson.databind.*;
+import org.apache.camel.*;
+import org.apache.camel.support.*;
+import org.mifos.connector.common.gsma.dto.*;
+import org.springframework.beans.factory.annotation.*;
+import org.springframework.web.bind.annotation.*;
+
+@RestController
+public class GSMADepositApiController implements GSMADepositApi {
+
+    @Autowired
+    private ProducerTemplate producerTemplate;
+
+    @Autowired
+    ObjectMapper objectMapper;
+
+    @Override
+    public RequestStateDTO gsmadeposit(String tenant, String correlationId, GSMATransaction requestBody) throws JsonProcessingException {
+        Exchange exchange = new DefaultExchange(producerTemplate.getCamelContext());
+        exchange.getIn().setBody(objectMapper.writeValueAsString(requestBody));
```

Comment:
Line 22 - 28 mostly duplicated in all API controllers.

Reply:
@avikganguly01 and @fynmanoj I have created builder pattern for headers and removed repeated code by adding it in utils.